diff -urN openh323gk.orig/Makefile openh323gk/Makefile
--- openh323gk.orig/Makefile	Tue Jul  9 18:27:03 2002
+++ openh323gk/Makefile	Tue Jul  9 18:35:44 2002
@@ -18,7 +18,7 @@
 
 H323_INCDIR	= ${OPENH323DIR}/include
 H323_LIBDIR	= ${OPENH323DIR}/lib
-H323_LIB	= h323_$(PLATFORM_TYPE)_$(OBJ_SUFFIX)
+H323_LIB	= openh323
 
 STDCCFLAGS := -DLDAP_HAS_CACHE -I${H323_INCDIR} -DPTRACING#-DPASN_NOPRINT
 #-DLDAP_HAS_CACHE : do not use ldap-cache (alpha)
