diff -urN openh323gk.orig/ldap/src/Makefile openh323gk/ldap/src/Makefile
--- openh323gk.orig/ldap/src/Makefile	Tue Jul  9 18:27:03 2002
+++ openh323gk/ldap/src/Makefile	Tue Jul  9 18:32:34 2002
@@ -10,7 +10,7 @@
 LDAP_INCLUDEDIR = ../include
 
 
-include $(PWLIBDIR)/make/unix.mak
+include $(PWLIBDIR)/share/pwlib/unix.mak
 
 LDAPAPI_BASE = ldapapi_$(PLATFORM_TYPE)_$(OBJ_SUFFIX)
 LDAPAPI_FILE = lib$(LDAPAPI_BASE).$(LIB_SUFFIX)
@@ -40,15 +40,15 @@
 
 SOURCES = $(COMMON_SOURCES)
 
-include $(PWLIBDIR)/make/unix.mak
-include $(PWLIBDIR)/make/ptlib.mak
-include $(PWLIBDIR)/make/lib.mak
+include $(PWLIBDIR)/share/pwlib/unix.mak
+include $(PWLIBDIR)/share/pwlib/ptlib.mak
+include $(PWLIBDIR)/share/pwlib/lib.mak
 
 ifeq (,$(wildcard $(OPENH323DIR)/include/ldap.h))
 STDCCFLAGS += -DOPENH323LIB_HAS_LDAP_ASN_H
 endif
 
-STDCCFLAGS += -I../include -I. -I$(OPENH323DIR)/include
+STDCCFLAGS += -I../include -I. -I$(OPENH323DIR)/include/openh323
 
 ifndef DEBUG
 	STDCCFLAGS += -fpic
