--- openh323gk/Makefile.orig	Sun Nov 17 14:20:40 2002
+++ openh323gk/Makefile	Sun Nov 17 14:31:45 2002
@@ -42,7 +42,7 @@
 XID=$$Id
 GKGVS="@(\#) $(XID): $(VERSION_STATUS) of "$(PROGRAMMNAME)" v$(VERSION_MAJOR).$(VERSION_MINOR) build\#$(VERSION_BUILD) by "${MANUFACTURER}" at " __DATE__ " "  __TIME__ " $$"
 
-H323_INCDIR = ${OPENH323DIR}/include
+H323_INCDIR = ${OPENH323DIR}/include/openh323
 H323_LIBDIR = ${OPENH323DIR}/lib
 H323_LIB    = h323_$(PLATFORM_TYPE)_$(OBJ_SUFFIX)
 
